pkg/handler/loki.go

```diff
@@ -115,6 +116,7 @@ func fetchSingle(lokiClient httpclient.Caller, flowsURL string, merger loki.Merg
 	}
 	var qr model.QueryResponse
 	if err := json.Unmarshal(resp, &qr); err != nil {
+		hlog.Errorf("Cannot unmarshal (%v). Response was: %s", err, string(resp))
```

Comment:
Another form is `hlog.WithError(err).Errorf("cannot unmarshal %v", string(resp))` but yours it's also fine.